app/views/carts/show.html.erb

```diff
@@ -1,9 +1,18 @@
-<%  @items.each do |item| %>
+<% @items.each do |item| %>
+<div class="item-<%= item.id %>">
 <%= image_tag(item.image, class: "item_thumbnail") %>
 <h5><%= item.title %></h5>
 <p><%= item.description %></p>
 <p>$<%= item.price %></p>
+<%= form_tag(cart_path, method: :patch) do %>
+  <%= hidden_field_tag(:item_id, item.id) %>
+  <%= label_tag("Quantity") %>
+  <%= text_field_tag("Quantity", "#{@cart.quantity(item)}") %>
```

Comment:
I think it was a great idea to have the individual form on the page. It should make for an elegant user experience. The hidden field tag passing the item to the params is a great way to do this and make the quantity update doable.